anonymousIdOptions added in JS SDK (#519)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/javascript-sdk-enhancements.mdx

<div class="warningBlock">
  

>   <strong>This feature is deprecated and RudderStack no longer provides its support</strong>.

Reviewer: ```suggestion
  <strong>This feature is deprecated and no longer supported by RudderStack</strong>.
```

Reviewer: Reads better, IMO.
